__tests__/src/packages/imperative/plugins/suites/UpdatePlugins.ts

```diff
@@ -12,15 +12,23 @@
 import * as T from "../../../../../src/TestUtil";
 import { cliBin } from "../PluginManagementFacility.spec";
 import { join } from "path";
+import { SetupTestEnvironment } from "../../../../../__src__/environment/SetupTestEnvironment";
+import { existsSync } from "fs";
+import { execSync } from "child_process";
 
 describe("Update plugin", () => {
     const testPluginDir = join(__dirname, "../test_plugins");
+    let envNpmRegistry;
 
     const removeNewline = (str: string): string => {
         str = str.replace(/\r?\n|\r/g, " ");
         return str;
     };
 
+    beforeAll(() => {
+        envNpmRegistry = execSync("npm config get registry").toString().trim();
+    });
```

Comment:
Is this `beforeAll` needed? I don't see `envNpmRegistry` used anywhere later in the file.